.github/workflows/prune.yml

```diff
@@ -49,8 +49,8 @@ jobs:
           # replace all whitespaces with newline (required for sort) and then sort all release versions while removing any duplicates
           all_release_versions_sorted=$(echo "${all_release_versions[@]}" | tr ' ' '\n'| sort -u )
           
-          # replace the newlines with commas and then remove the trailing comma
-          all_release_versions=$(echo "${all_release_versions_sorted[@]}" | tr '\n' ',' | sed 's/,$//')
+          # remove any blank rows, replace the newlines with commas, then remove the trailing comma
```

Comment:
I: ` sed -r '/^\s*$/d'` command checks for any rows that only contain whitespace and deletes it